test/JDBC/expected/pivot-vu-verify.out

```diff
@@ -1124,7 +1233,394 @@ int#!#varchar#!#varchar#!#varchar
+) AS p1
+RIGHT JOIN
+(
+    SELECT TOP 10 EmployeeID, [7] AS STORE7, [8] AS STORE8, [9] AS STORE9, [10] AS STORE10
+    FROM
+    (
+        SELECT EmployeeID, ItemID, StoreID
+        FROM StoreReceipt
+    )AS srctable
+    PIVOT (
+        COUNT (srctable.ItemID)
+        FOR StoreID IN ([7], [8], [9], [10])
+    ) AS pvt where EmployeeID > 245
+) AS p2
+ON p1.EmployeeID = p2.EmployeeID ORDER BY 1
+GO
+~~START~~
+int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#246#!#0#!#1#!#0#!#0
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#247#!#0#!#0#!#2#!#0
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#248#!#0#!#0#!#0#!#1
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#249#!#1#!#1#!#0#!#0
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#250#!#0#!#0#!#0#!#1
+251#!#1#!#1#!#1#!#0#!#0#!#251#!#0#!#0#!#0#!#0
+252#!#0#!#0#!#0#!#0#!#0#!#252#!#1#!#0#!#0#!#0
+253#!#1#!#0#!#0#!#0#!#0#!#253#!#1#!#0#!#1#!#0
+254#!#0#!#1#!#0#!#0#!#0#!#254#!#0#!#0#!#0#!#0
+255#!#0#!#1#!#0#!#0#!#0#!#255#!#0#!#2#!#0#!#0
+~~END~~
+
+
+-- FULL JOIN TEST
+SELECT * FROM
+(
+    SELECT TOP 10 EmployeeID, [2] AS STORE2, [3] AS STORE3, [4] AS STORE4, [5] AS STORE5, [6] AS STORE6
+    FROM
+    (
+        SELECT EmployeeID, ItemID, StoreID
+        FROM StoreReceipt
+    )AS srctable
+    PIVOT (
+        COUNT (srctable.ItemID)
+        FOR StoreID IN ([2], [3], [4], [5], [6])
+    ) AS pvt where EmployeeID > 250
+) AS p1
+FULL JOIN
+(
+    SELECT TOP 10 EmployeeID, [7] AS STORE7, [8] AS STORE8, [9] AS STORE9, [10] AS STORE10
+    FROM
+    (
+        SELECT EmployeeID, ItemID, StoreID
+        FROM StoreReceipt
+    )AS srctable
+    PIVOT (
+        COUNT (srctable.ItemID)
+        FOR StoreID IN ([7], [8], [9], [10])
+    ) AS pvt where EmployeeID > 245
+) AS p2
+ON p1.EmployeeID = p2.EmployeeID ORDER BY 1
+GO
+~~START~~
+int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int#!#int
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#250#!#0#!#0#!#0#!#1
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#247#!#0#!#0#!#2#!#0
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#246#!#0#!#1#!#0#!#0
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#248#!#0#!#0#!#0#!#1
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#249#!#1#!#1#!#0#!#0
+251#!#1#!#1#!#1#!#0#!#0#!#251#!#0#!#0#!#0#!#0
+252#!#0#!#0#!#0#!#0#!#0#!#252#!#1#!#0#!#0#!#0
+253#!#1#!#0#!#0#!#0#!#0#!#253#!#1#!#0#!#1#!#0
+254#!#0#!#1#!#0#!#0#!#0#!#254#!#0#!#0#!#0#!#0
+255#!#0#!#1#!#0#!#0#!#0#!#255#!#0#!#2#!#0#!#0
+256#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+258#!#0#!#0#!#0#!#0#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+259#!#0#!#1#!#0#!#1#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+260#!#0#!#0#!#0#!#0#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+261#!#0#!#0#!#0#!#0#!#0#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+~~END~~
+
+
+-- COMMA JOIN TEST
```

Comment:
If you are testing all join types here - we also have CROSS JOIN

Reply:
Added CROSS JOIN tests (with or without CROSS JOIN keyword) and verified results